java/feature-collection-layer-query/README.md

```diff
@@ -1,21 +1,40 @@
-# Feature Collection Layer (Query)
-Create a feature collection layer to show a query result from a service feature table. The feature collection is then displayed on a map with a feature collection layer.
+# Feature collection layer (query)
 
-![Feature Collection Layer (Query) App](feature-collection-layer-query.png)
+Create a feature collection layer to show a query result from a service feature table.
+
+![Image of feature collection layer](feature-collection-layer-query.png)
+
+## Use case
+
+Consider a national wireless telecommunications company who wants to improve connectivity within areas of poor coverage. Their cell towers, cell tower radii and state boundaries data are held within service feature tables, which can be queried for states that don't meet a certain threshold of coverage. The query result can be used to create and display a single feature collection layer for analysis of areas requiring improved coverage.
 
 ## How to use the sample
-Simply run the app.
+
+When launched, this sample displays a map with point features as a feature collection layer. Pan and zoom to explore the map.
 
 ## How it works
-A query is performed using the `.queryFeaturesAsync(queryParameters)` method on `FeatureTable`. The result of the query is used to instantiate an `FeatureCollectionTable`. The table is used to instantiate an `FeatureCollection` which is then use to initialize a `FeatureCollectionLayer`. The layer is then displayed on the map by adding it to the operational layers array.
+
+1. Perform a query using the `queryFeaturesAsync()` method on a `ServiceFeatureTable`.
+2. Instantiate a `FeatureCollectionTable` with the result of the query.
+3. Create a `FeatureCollection` and add the feature collection table to it.
+4. Create a `FeatureCollectionLayer` from the feature collection.
+5. Add the feature collection layer to the map's `operationalLayers` list.
 
 ## Relevant API
+
 * FeatureCollection
 * FeatureCollectionLayer
 * FeatureCollectionTable
+* FeatureLayer
 * FeatureQueryResult
+* FeatureTable
 * QueryParameters
 * ServiceFeatureTable
 
-#### Tags
-Search and Query
+## About the data
```

Comment:
Noticed from doing the Java SE readmes: there is an erroneous "About the Data" section in here. It refers to wildfires, whereas in our sample the data is hard coded. We should just take the About the Data section out in this case.